src/components/BalanceList.js

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import Balance from './Balance.js';
+
+export default function BalanceList({
+  expenses,
+  setExpenses,
+  setEditExpense,
+  members,
+  expenseFormatter,
+}) {
+  const totalExpenses = Number(
+    expenses
+      .map(expense => expense.amount)
+      .reduce((prev, curr) => +prev + +curr, 0)
+    //The unary plus operator (+prev) converts the strings to numbers
+  );
+
+  return (
+    <Wrapper>
+      <TotalExpenses>
+        <p>Total expenses:</p>
+        <p>{expenseFormatter(totalExpenses)}</p>
+      </TotalExpenses>
+      <Scrollarea>
+        {members.map((member, index) => (
+          <Balance
+            key={index}
+            member={member}
+            expenses={expenses}
+            expenseFormatter={expenseFormatter}
+          />
+        ))}
+      </Scrollarea>
+    </Wrapper>
+  );
+
+  function expenseFormatter(num) {
+    //puts the expense into a 1,234,567.89€ format
+    let p = Number(num).toFixed(2).split('.');
+    return (
+      (p[0].split('')[0] === '-' ? '-' : '') +
+      p[0]
+        .split('')
+        .reverse()
+        .reduce(function (acc, num, i, orig) {
+          return num === '-' ? acc : num + (i && !(i % 3) ? ',' : '') + acc;
+        }, '') +
+      '.' +
+      p[1] +
+      ' €'
+    );
+  }
+}
```

Comment:
This is the same function like in the App.js when I am right. Isn´t it possible to handle it as prop to the BalanceList to reduce code?

Reply:
Pulled it up and forgot to delete it here. The App.js one was the only one in use already.